packages/core/src/jwt.ts

```diff
@@ -117,21 +117,32 @@ export class Jwt<
    * Verify the JWT using the provided verifier function.
    * It checks the signature and validates the iat, nbf, and exp claims if they are present.
    * @param verifier
-   * @param currentDate
+   * @param currentDate current time in seconds since epoch
+   * @param skew allowed skew in seconds for the current time in milliseconds. Positive value that will lower the iat and nbf checks, and increase the exp check.
    * @returns
    */
   public async verify(
     verifier: Verifier,
     currentDate = Math.floor(Date.now() / 1000),
+    skew: number = 0,
```

Comment:
Would it make sense to:
- rename `skew` to `skewSeconds` (it can be hard to know in JS whether we're talking about seconds or milliseconds)
- make the `currentDate` and `skew` an object called `options`, this way adding new options will be simpler, and won't require a lot of positional arguments (which are harder to understand when just looking at the code).

```ts
verify(jwt, new Date(), 100)
```

vs

```ts
verify(jwt, {
  currentDate: new Date(), 
  skewSeconds: 100
})
```

Reply:
You are right, I moved it to an option object and made both values in seconds.